src/docs/operator-guides/edit-seo-and-social-tags/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to edit SEO & social tags in Pages
+slug: how-to-edit-seo-and-social-tags-in-pages
+updated: 2023-02-08
+category: operator-guides
+ingress: Change SEO and social tags in Pages
+published: true
+---
+
+Meta tags are a fundamental part of any static content page. They
+determine how social media platforms, browsers and search engines will
+display your marketplace. Google, for example, uses your SEO metatags to
+populate their search results page.
+
+## Edit your meta tags
```

Comment:
Also here and everywhere below I would steer away from using the word "tag". All instances of "tag" should be removed. Let's call this "metadata".

Reply:
Juho I agree with the title, maybe it is metadata, but aren't these elements also tags? I googled quickly and there are many sites that still call them tags. At least MOZ calls them Title tags ot Meta tags. I don't know if calling them meta data instead of meta tags or metadata tags would make this article easier to understand for the general people. MOZ even says: You may also see title tags referred to as "meta titles" or "meta title tags" in some cases. While controversial with some, either usage is typically fine.

Reply:
@Luis-RG are you sure the social title, description and image are also tags?

Reply:
social title, description and social media image can be called open graph tags or open graph meta tags but they're still technically <meta> tags.

Reply:
@Kusti I'm not an expert, so can never be sure, but I googled and most people still talk about meta tags online instead of metadata. In [Google](https://developers.google.com/search/docs/crawling-indexing/special-tags), [Facebook](https://developers.facebook.com/docs/marketing-api/catalog/guides/metadata-tags/), [Twitter](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup) and [Linkedin](https://www.linkedin.com/pulse/html-headers-og-meta-tags-david-polcari/) dev documentation they still talk about Tags. So I think I will leave the title as suggested by you but keep in the content calling them Tags